### PR TITLE
Load data before dom is ready

### LINKED ISSUE #57: Start loading path coverage earlier
We probably don't need to wait for onLoad in order to start loading the data.

### PR DESCRIPTION
**Pull request by La0**
:zap: 

Fixes #57

### CODE REVIEW

File: src/common.js [hunk 1]
```diff
@@ -6,18 +6,31 @@ function assert(condition, message) {
   }
 }
 
-function windowLoaded() {
-  return new Promise(resolve => window.onload = resolve);
+function domContentLoaded() {
+  return new Promise(resolve => {
```
Comment (marco-c) [inline src/common.js]: Nit: you can remove the `{`, `}` and `;`

File: src/common.js [hunk 2]
```diff
@@ -241,20 +254,30 @@ function navbar(path, revision) {
 }
 
 
-// Display message as main output
+// Display helpers
+function canDisplay() {
```
Comment (marco-c) [inline src/common.js:256]: Why do we need this? Don't we always call the display functions when everything is ready?

**Review by marco-c — APPROVED** [review]

File: src/common.js (hunk 1 above)
Comment (La0) [reply src/common.js]: You are such a l33t Javascript overlord.

File: src/common.js (hunk 2 above)
Comment (La0) [reply src/common.js:256]: This is for [this execution of hide & message](https://github.com/La0/code-coverage-reports/blob/fix-57/src/index.js#L184) which is in `load`: it's called when dom is not yet ready on first load, but is nice to have for later calls, as this loading time is a bit long.

**Pull request merged**